extensions/common/monitor/monitor-jdk-logger/src/test/java/org/eclipse/edc/monitor/logger/PrefixMonitorTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ *  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH -  initial implementation
+ *
+ */
+
+package org.eclipse.edc.monitor.logger;
+
+import org.eclipse.edc.spi.monitor.Monitor;
+import org.eclipse.edc.spi.monitor.PrefixMonitor;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+
+import java.util.function.Supplier;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.verify;
+
+public class PrefixMonitorTest {
```

Comment:
This test class is too complicated and opaque. It should be apparent what is being asserted by quickly looking at the individual test methods. Please:

1. Replace the use of captors with argument matchers and strict mocks. This will make the test easier to read and less verbose. 
2. Inline the assert methods in all test methods rather than having two separate private methods, one of which calls a supplier and delegates to the other